docker-compose.yml

```diff
@@ -144,8 +146,19 @@ services:
             - SCREEN_DEPTH=24
         depends_on:
             - seleniumhub
+
+    docker:
+        build: docker
         networks:
             - omero-network
+        volumes:
+            - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
Will this have any impact/constraint on the deployment of devspaces that did not exist before?

Reply:
Not really. I haven't tested on mac, but openstack & the orcas should be fine. It *is* a security issue and it *does* impact _distributed_ installations (kubernetes, docker swarm) so we will have to come back to it in the future.